distrobox-init

```diff
@@ -385,7 +385,7 @@ passwd --delete root
 if [ -d "/etc/skel" ]; then
 	skel_files="$(find /etc/skel/ -type f || :)"
 	for skel_file in ${skel_files}; do
-		if [ ! -f "${container_user_home}/$(basename "${skel_file}")" ]; then
+		if [ ! -f "${container_user_home}/$(basename "${skel_file}")" ] && [ !-L "${container_user_home}/$(basename "${skel_file}")" ]; then
```

Comment:
`distrobox-init:388:73: error: You are missing a required space here. [SC1035]`

Need a space between the `!` and `-L`